### ISSUE TITLE
ViewPropTypes warnings

### ISSUE THREAD

**Issue by rick427** [lottie-react-native/lottie-react-native#858]
Hi, after installing this package (v5.0.1) in my RN project (v0.68.0), i seem to be getting this warning **ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'** all over the place. How do i clear these warning pls. It's super annoying to look at.

Thanks

**Comment by jalinegm** [comment]
same here

**Comment by siderakis** [comment]
```
import { LogBox } from 'react-native';

LogBox.ignoreLogs(['ViewPropTypes will be removed from React Native']);

```

**Comment by PAAPII10** [comment]
> Call this in index.js or App.tsx/js etc.
> 
> ```
> import { LogBox } from 'react-native';
> 
> LogBox.ignoreLogs(['ViewPropTypes will be removed from React Native']);
> ```

The app gets crash... it is not just a warning

**Comment by axelhildingson** [comment]
There is a fix for it (https://github.com/lottie-react-native/lottie-react-native/commit/9dc4b5720a512a66d5f16533abde7b348a6ca716) but the package has been released jet.

@emilioicai is it possible to release a new version of the package since expo just release 45 and it uses RN 0.68.1

**Comment by emilioicai** [comment]
I'm planning on releasing 6.0.0 this week. It's a major version change so it's taking longer than normal. Apologies for that

**Comment by KieQuang** [comment]
@emilioicai i'm still waiting. Thank.

**Comment by AxanIqbal** [comment]
is there any patch-package or quick fix of that ? having error about too many re-renders was working on previous expo 44

**Comment by thespacemanatee** [comment]
It's quite annoying for us as well and hurting productivity big time. Simply using patch-package is not working either as the error comes from `/lib`. Why not release a minor first? @emilioicai

**Comment by emilioicai** [comment]
Since there were some complaints I just released v5.1.0 containing this fix and several others. Integration with C++/WinRT will need to wait as testing was not completed.
